linode/manager
# refactor: [M3-6382] - MUI v5 - `Components > Radio`

## Description 📝

- Style migration for `<Radio />`

## Major Changes 🔄
- Moved styles from `Radio.tsx` into the theme
- Updates Storybook story to CSF 3 file with a `mdx` file to retain docs
- Removes `core/Radio.ts` file and `index.ts` file so there is only one source of truth for the `<Radio />`

## Preview 📷

![Screenshot 2023-05-26 at 4 14 31 PM](https://github.com/linode/manager/assets/115251059/16c3344d-51df-4190-acc7-18b420d336e1)


## How to test 🧪
- View the Storybook stories
  - Check the Stories and make sure they look like what you'd expect
  - Check for styling differences between local and https://design.linode.com
- Check the app for style regressions wherever radios are used
  - Database Create
  - Profile Settings
  - Add IP Address Drawer
